carsus/io/kurucz/gfall.py

```diff
@@ -395,7 +396,7 @@ def to_hdf(self, fname):
 
 class GFALLIngester(object):
     """
-        Class for ingesting data from kurucz dfall files
```

Comment:
I think this was a typing error? I changed it to gfall instead of dfall

Reply:
thanks!